thread-archiver

```diff
@@ -37,6 +37,7 @@ Options:
   --follow-children              Follow threads linked in downloaded threads
   --follow-to-other-boards       Follow linked threads, even if from other boards
   --silent                       Suppresses mundane printouts, prints what's important
+  --jinja2                       Use new jinja2 templating instead of downloading HTML
```

Comment:
I'd probably name this argument more along the lines of `use-template` rather than `jinja2`, since that's related more to the feature it provides rather than the name of the library doing it.